portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,8 +40,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      String userInput = request.getParameter("user-comment");
-      messages.add(userInput);
```

Comment:
You should probably leave these until you get the doGet method hooked up.  Otherwise this change will break the comment feature.  It's good to add incremental steps that do nothing (e.g. storing the comments in a database but never reading them), but you want to avoid breaking features in the process.  Otherwise this point in the change history will be broken